core/feed.py

```diff
@@ -50,6 +50,13 @@ def update_feed(feed_id):
             f.update_status("OK")
         else:
             logging.debug("Feed {} has been disabled".format(f.name))
+    except GenericYetiInfo as e:
+        msg = "INFO updating feed: {}".format(e)
+        logging.info(msg)
+        f.update_status(msg)
+        f.modify(lock=False)
+        return False
```

Comment:
Why return `False` ?

Reply:
Bcz the original exception was also returning False